src/server/scripts/Outland/CoilfangReservoir/SerpentShrine/boss_leotheras_the_blind.cpp

```diff
@@ -217,7 +217,8 @@ class boss_leotheras_the_blind : public CreatureScript
             IsFinalForm = false;
             NeedThreatReset = false;
             EnrageUsed = false;
-            memset(InnderDemon, 0, sizeof(InnderDemon));
+            for (ObjectGuid& guid : InnderDemon)
+                guid.Clear();
```

Comment:
This is redundant,`ObjectGuid` zero-initializes

Reply:
Correct me if I'm wrong, but isn't `Initialize()` called from `Reset()` as well? The zero-initialization shouldn't help in that case.

Reply:
it is, this change is required and looks good.